Get Started guide on website for BonFHIR + React (#122)
bonfhir/bonfhir: docs/website/docs/build-a-fhir-app-with-react/display-fhir-data.md

2. In the newly created Vite app, open `src/pages/Home.tsx`

3. To fetch a Patient with `useFhirRead`, indicate that the type is `"Patient"` and the ID is what you copied in step 1. For example: `"afb2bbf9-872c-47a9-9b31-2a737ed65f0b"`

   ```tsx
   const patientQuery = useFhirRead(
     "Patient",
     "afb2bbf9-872c-47a9-9b31-2a737ed65f0b",
   );
   ```

4. Display the patient's name with `<FhirValue />` where the type is `"HumanName"`

   ```tsx
   <FhirValue type="HumanName" value={patientQuery.data?.name} />
   ```

5. Wrap the value inside the `<FhirQueryLoader />` and pass `patient` directly into the value component. The query loader will handle loading and error states with automatic retries. To preview this functionality, replace the patient ID with a value that does not exist in the database.

   ```tsx
   return (
      <FhirQueryLoader query={patientQuery}>
         {(patient) => (
            <FhirValue type="HumanName" value={patient.name} />
         )}
      <FhirQueryLoader />
   );
   ```

> 6. BonFHIR comes with [Mantine](https://mantine.dev/) components ready to use for developing the application. We can now add additional information about the patient including their birthday, addresses and contact points. `FhirValue` will take care of all data formatting for us.

Reviewer: ```suggestion
6. BonFHIR components render using [Mantine](https://mantine.dev/) components - so you can mix & match with all Mantine components for layout and data presentation. We can now add additional information about the patient including their birthday, addresses and contact points. `FhirValue` will take care of all data formatting for us. Don't hesitate to play with options as well to explore formatting capabilities.
```

   ```tsx
   <Group>
      <Text fw={600}>Birthday: </Text>

>       <FhirValue type="date" value={patient.birthDate} />

Reviewer: ```suggestion
      <FhirValue type="date" value={patient.birthDate} options={{ dateStyle: "full" }} />
```

Author: This is great 👍 A good opportunity to showcase the power of the FHIR value formatters. Thank you!

   </Group>
   <Group>
      <Text fw={600}>Address: </Text>
      <FhirValue type="Address" value={patient.address} />
   </Group>
   <Group>
      <Text fw={600}>Contact: </Text>
      <FhirValue type="ContactPoint" value={patient.telecom} />
   </Group>
   ```

## Final result

Here is the output of this example with patient information displayed on a single card:

![Example](../../static/img/docs/display-fhir-data-example.png)

```tsx title="src/pages/Home.tsx"
import { useFhirRead } from "@bonfhir/query/r4b";
import { FhirQueryLoader, FhirValue } from "@bonfhir/react/r4b";
import { Group, Paper, Stack, Text } from "@mantine/core";

export default function Home() {
  const patientQuery = useFhirRead(
    "Patient",
    "afb2bbf9-872c-47a9-9b31-2a737ed65f0b",
  );

  return (
    <Paper p="xl">
      <Paper shadow="xs" p="xl">
        <FhirQueryLoader query={patientQuery}>
          {(patient) => (
            <Stack gap="sm">
              <Text size="xl">
                <FhirValue type="HumanName" value={patient.name} />
              </Text>
              <Group>
                <Text fw={600}>Birthday: </Text>

>                 <FhirValue type="date" value={patient.birthDate} />

Reviewer: ```suggestion
                <FhirValue type="date" value={patient.birthDate} options={{ dateStyle: "full" }} />
```
